### DISCUSSION TITLE
Setting error status after so many send attempts.

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#4795] [Q&A] [answered]
I tried putting this code in the response transformer but it never seems to even go in there

`logger.error('in response tranformer');

if (connectorMessage.getSendAttempts() >= 5) 
{
	responseStatus = ERROR;
}`

I want the message to error after 5 attempts. Is there another way to do this?

**Comment by pacmano1** [comment, marked as answer]
Is your response data type set to raw? Otherwise won't fire.

**Comment by rutledj** [comment]
Yep. that worked. Thx.
